CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
     - `sdk.users.add_role()`
     - `sdk.users.remove_role()`
 
+- `__eq__` and `__hash__` methods to the `py42.sdk.queries.query_filter.QueryFilter` class to enable easier comparison of filters
```

Comment:
We should also mention the sorted change here, no?  Results that were not sorted before will now be sorted. This could break some unit tests. Also, would let users know they don't have to manually sort them if they were before.

Reply:
yes, in fact this change caused our own unit tests to need to be changed, so it should be called out.

Reply:
good call, I'll note it